scripts/gatsby-plugin-checker/index.js

```diff
@@ -64,6 +64,60 @@ const filterNotNotified = (packages, plugins) => {
   return packages.filter(pkg => notified.indexOf(pkg.name) < 0)
 }
 
+const removePackagesWithoutRepository = packages =>
+  packages.filter(p => hasRepository(p))
+
+const hasRepository = packageToCheck => {
+  if (packageToCheck.links.repository) {
+    return true
+  }
+  return false
+}
+
+const removePackagesWithBadRepoLinks = async packages => {
```

Comment:
I'm not sure if we should remove these packages. The plugins still work and are on npm. It's not required to have a github repo.